crowbar_framework/app/views/setups/show.html.haml

```diff
@@ -0,0 +1,66 @@
+%section
+  .container
+    %p Step 3: This needs to be added to the beginning page of the installer, tweaks might be needed
+    .row
+      .panel.panel-default
+        .panel-heading
+          = button_to "Continue Upgrade", backups_path, class: "btn btn-primary", method: :get
+        .panel-body
+          %p ...this is where the rest of the first page of the current installer should begin...or maybe not, who knows.
+          %p Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+%section
+  .container
+    %p Step 4:
+    .row
+      .panel.panel-default
+        .panel-heading
+          %h4 Missing Repositories
+        .panel-body
+          %p The software repository for "SUSE Linux Enterprise High Availability Extension" and "SUSE Linux Enterprise Storage" are missing. Please configure these by hand and click recheck below when the repositories are available.
+          = button_to "Recheck Repositories", repositories_path, class: "btn btn-primary center-block", method: :get
+%section
+  .container
+    %p Step 5:
+    .row
+      .panel.panel-default
+        .panel-heading
+          %h4 Stop OpenStack Services
+        .panel-body
+          %p Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          = button_to "Continue Upgrade Process", stop_setup_path, class: "btn btn-success center-block", method: :get
+%section
+  .container
+    %p Step 6:
+    .row
+      .panel.panel-default
+        .panel-heading
+          %h4 Data Backup
+        .panel-body
+          %p Now is the time to backup all the other data in your system.
+          %p Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          %button.btn.btn-success.center-block{:type => "button"} Continue Upgrade Process
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace